packages/editor-ui/src/views/TestDefinition/TestDefinitionListView.vue

```diff
@@ -1,22 +1,59 @@
 <script setup lang="ts">
 import { ref, computed, onMounted } from 'vue';
 import { useRouter } from 'vue-router';
-import { VIEWS } from '@/constants';
+import { MODAL_CONFIRM, VIEWS } from '@/constants';
 import { useTestDefinitionStore } from '@/stores/testDefinition.store.ee';
 import { useToast } from '@/composables/useToast';
 import { useI18n } from '@/composables/useI18n';
 import EmptyState from '@/components/TestDefinition/ListDefinition/EmptyState.vue';
 import TestsList from '@/components/TestDefinition/ListDefinition/TestsList.vue';
-import type { TestExecution, TestListItem } from '@/components/TestDefinition/types';
+import type {
+	TestExecution,
+	TestItemAction,
+	TestListItem,
+} from '@/components/TestDefinition/types';
 import { useAnnotationTagsStore } from '@/stores/tags.store';
 import type { TestDefinitionRecord } from '@/api/testDefinition.ee';
+import { useMessage } from '@/composables/useMessage';
 
 const router = useRouter();
 const tagsStore = useAnnotationTagsStore();
 const testDefinitionStore = useTestDefinitionStore();
 const isLoading = ref(false);
 const toast = useToast();
 const locale = useI18n();
+const { confirm } = useMessage();
+
+const actions = computed<TestItemAction[]>(() => [
+	{
+		icon: 'play',
+		id: 'run',
+		event: onRunTest,
+		disabled: isRunDisabled,
+		tooltip: (testId) =>
+			isRunDisabled(testId)
+				? getDisabledRunTooltip(testId)
+				: locale.baseText('testDefinition.runTest'),
```

Comment:
Was cancel action button lost after merging master?

Reply:
Does it make sense to hide the play button instead of disabling it?

Reply:
It's following the product specs :D I also like disabled version better because we cant let user know why it's disabled.